src/read.rs

```diff
@@ -686,10 +691,28 @@ impl<R: Read + Seek> ZipArchive<R> {
                 if let Some(p) = outpath.parent() {
                     Self::make_writable_dir_all(p)?;
                 }
+                /* FIXME: what bug does this code fix?
+                 * https://github.com/zip-rs/zip2/commit/8715d936cbd3e0fdc2e778b70d8a02e2dfc16fc2
+                 * doesn't specify! */
```

Comment:
```suggestion
                // Extract symlinks as symlinks
```
The bug was that symlinks were extracted as normal files.